docs/installing.rst

```diff
@@ -1,18 +1,19 @@
 Installation
 ============
 
-Install the package (or add it to your ``requirements.txt`` file):
+How to install and use the theme
+--------------------------------
+
+Install the ``sphinx_rtd_theme`` package (or add it to your ``requirements.txt`` file):
 
 .. code:: console
 
     $ pip install sphinx_rtd_theme
 
-In your ``conf.py`` file:
+In your Sphinx project's ``conf.py`` file, add ``sphinx_rtd_theme`` to the list of enabled extensions and as the active theme:
 
 .. code:: python
 
-    import sphinx_rtd_theme
-
```

Comment:
By importing this module, nothing happens. It's not used in `conf.py` and nothing is executed in the module's `__init__.py`.

I'm 99.9% sure that this is here for some kind of legacy reason.

Reply:
I also tested it multiple times. I haven't used the theme recently with this line of code.

Reply:
I believe you are right, and import here was only required because the conf.py used to require manual manipulation of the theme path.